navigation/ca_collaborative_move_base/config/costmap_common_params.yaml

```diff
@@ -0,0 +1,27 @@
+robot_base_frame: unit_box
+
+transform_tolerance: 0.5
+
+robot_radius: 0.18
```

Comment:
Don't we have another parameter to consider a square robot?

Reply:
used unit_box side size 1, and radius for inscripted square in a circle is 0.707